src/LODVisitor.cpp

```diff
@@ -113,24 +114,29 @@ bool testFrontFacing(glm::dvec3 const& camPos, PlanetParameters const* params,
           glm::dvec3(tbMax[0], tbMax[1], tbMax[2]), glm::dvec3(tbMin[0], tbMax[1], tbMax[2])}};
 
   // Simple ray-sphere intersection test for every corner point
-  for (std::size_t j = 0; j < tbPnts.size(); ++j) {
-    double     dRayLength = glm::length(tbPnts[j] - camPos);
-    glm::dvec3 vRayDir    = (tbPnts[j] - camPos) / dRayLength;
+  for (auto& tbPnt : tbPnts) {
```

Comment:
Can we use const& here?

```suggestion
  for (auto const& tbPnt : tbPnts) {
```